crates/levm/tests/tests.rs

```diff
@@ -1162,6 +1161,455 @@ fn mload_uninitialized_memory() {
     assert_eq!(memory_size, U256::from(96));
 }
 
+#[test]
+fn call_returns_if_bytecode_empty() {
+    let callee_bytecode = vec![].into();
+
+    let callee_address = Address::from_low_u64_be(U256::from(2).low_u64());
+    let callee_address_u256 = U256::from(2);
+    let callee_account = Account::new(U256::from(500000), callee_bytecode);
+
+    let caller_ops = vec![
+        Operation::Push32(U256::from(100_000)), // gas
+        Operation::Push32(callee_address_u256), // address
+        Operation::Push32(U256::zero()),        // value
+        Operation::Push32(U256::from(0)),       // args_offset
+        Operation::Push32(U256::from(0)),       // args_size
+        Operation::Push32(U256::from(0)),       // ret_offset
+        Operation::Push32(U256::from(32)),      // ret_size
+        Operation::Call,
+        Operation::Stop,
+    ];
+
+    let mut vm = new_vm_with_ops_addr_bal(
+        &caller_ops,
+        Address::from_low_u64_be(U256::from(1).low_u64()),
+        U256::zero(),
+    );
+
+    vm.add_account(callee_address, callee_account);
+    println!("to excec");
```

Comment:
```suggestion
```
I think we should delete the debug prints in tests

Reply:
done!